libs/networking/sources/DiffLogger.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+** EPITECH PROJECT, 2023
+** R-Type Networking
+** File description:
+** Diff Logger
+*/
+
+#include "DiffLogger.hpp"
+#include "Serialization/Deserializer.hpp"
+#include "Serialization/Serializer.hpp"
+#include "Snapshot.hpp"
+#include <iomanip>
+
+#ifdef _WIN32
+    #include <process.h>
+#else
+    #include <unistd.h>
+#endif
+
+net::system::DiffLogger::DiffLogger(engine::Registry &registry)
+    : _registry(registry)
+{
+    auto pid = getpid();
```

Comment:
'getpid' is deprecated: The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _getpid. See online help for details.

Reply:
_getpid c'est windows only donc c'est stupide

Reply:
idk, mais il me dit ça en tout cas